examples/signals_annotate/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/views"
+	clarifyx "github.com/clarify/clarify-go/x"
+)
+
+func main() {
+	// To select or publish signals, you must grant the integration access to
+	// the "admin" namespace in the Clarify admin panel.
+	creds, err := clarify.CredentialsFromFile("clarify-credentials.json")
+	if err != nil {
+		panic(err)
+	}
+
+	ctx := context.Background()
+	client := creds.Client(ctx)
+	xclient := clarifyx.Upgrade(client)
+
+	// For this example, the signals we want to select are created by the same
+	// integration that we are using to select them. Note that this isn't a
+	// requirement; for production cases, you may want this integration ID to be
+	// configured to be something else.
+	integrationID := creds.Integration
+	signalID := "cl1d4kobi2aq0ttkc4b0"
+
+	data := []views.SignalAnnotate{{
+		ID: signalID,
+		Annotations: map[string]string{
+			"my-new-annotation":                      "with-new-value",
+			"annotations-to-delete-has-empty-values": "",
+		},
```

Comment:
nit: Examples should, if possible, follow the theme of Arrested Development.

Reply:
This was originally started by either Bjarne or Bernt; I have just carried it forward...

Reply:
Done!